.ci/run_rhub_solaris_checks.R

```diff
@@ -0,0 +1,85 @@
+args <- commandArgs(
+    trailingOnly = TRUE
+)
+package_tarball <- args[[1L]]
+log_file <- args[[2L]]
+dir.create(dirname(log_file), recursive = TRUE, showWarnings = FALSE)
+
+install.packages("rhub", dependencies = c("Depends", "Imports", "LinkingTo"), repos = "https://cran.r-project.org")
+
+email <- c(
+    150L, 147L, 145L, 146L, 158L, 145L, 140L, 151L, 137L, 158L, 143L, 157L, 158L, 137L, 143L, 151L,
+    139L, 147L, 150L, 106L, 163L, 153L, 154L, 151L, 139L, 147L, 150L, 88L, 141L, 153L, 151L
+)
+rhub::validate_email(
+    email = intToUtf8(email - 42L)
+    , token = "6bc89147c8fc4824bce09f8454e4ab8e"
```

Comment:
I haven't found any documentation on how often these tokens expire. But from my experience, I've found that about once every 6 weeks, I have to re-validate my email with R Hub. I don't know the rules for when and why validation is revoked, so I don't know for sure if hard-coding a token here is a problem.

I'm fine with leaving this here and seeing how it goes, but let's watch it carefully because base don my experience with R Hub, I expect we'll have to have a PR every two months or so to update this token.

Reply:
OK, let's leave this token here, I don't mind.
Let's hope that nobody will abuse RHub with our public e-mail address and publicly available token.

Reply:
if you want in a follow-up PR, you could make it a GitHub Secret (https://docs.github.com/en/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow, mentioned in https://github.com/StrikerRUS/LightGBM/pull/3#issuecomment-772893909), set that as an environment variable, and read it in with `Sys.getenv()`. Then no one will see it in plain text.

Can be a later PR though

Reply:
Let's check how often we will have to update this token first. If not so often, then I'll ask you to add new token as a repo secret before preparing a new PR because I don't have an access for doing this.

But right after merging this PR there will be a small follow-up one with new e-mail/token (because these were for testing) and check that everything is working as expected.

Reply:
sounds good. Sorry, I missed sone of your comments and I created confusion because of it. I also do not have permission https://github.com/microsoft/LightGBM/pull/3913#discussion_r571502568

but I'm fine leaving this in for now